R/Artifact.R

```diff
@@ -15,8 +15,10 @@ py_to_r.lamindb.models.artifact.Artifact <- function(x) { # nolint object_length
       open = function(mode = "r", is_run_input = NULL, ...) {
         artifact_open(self, mode = mode, is_run_input = is_run_input, ...)
       },
-      view_lineage = function(with_children = TRUE) {
-        view_lineage()
+      view_lineage = function(with_children = TRUE, return_graph = FALSE) {
```

Comment:
Why are you setting this to `False`? I thought the point was to take this and visualize it? 🤔